azure_functions_worker/bindings/datumdef.py

```diff
@@ -192,7 +192,7 @@ def datum_as_proto(datum: Datum) -> protos.TypedData:
                 k: v.value
                 for k, v in datum.value['headers'].items()
             },
-            cookies=parse_to_rpc_http_cookie_list(datum.value['cookies']),
+            cookies=parse_to_rpc_http_cookie_list(datum.value.get('cookies')),
```

Comment:
Shouldn't we do this for all the values in the method parse_to_rpc_http_cookie_list?

Reply:
other fields are always set in older versions also so no need theoretically